utils/docker-compose/node-sdk/invoke.js

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Couldn't we make these scripts directly executable via a #!/usr/bin/node?
(The same comment of course applies to the other js scripts ...)

Reply:
For now I think I am going to leave them as node scripts and not executables. Locally on my machine node is installed in a different location so my guess is that is not a universally accepted location of node? But I do like the idea of running them like an executable eventually.